services/QuillLMS/app/services/classlink_integration/classrooms_fetcher.rb

```diff
@@ -5,20 +5,36 @@ class ClassroomsFetcher < BaseFetcher
     RESOURCE = 'classes'
     QUERY_STRING = "?filter=status='active'"
 
+    GRADE_ATTR = 'grades'
     SOURCED_ID_ATTR = 'sourcedId'
     TITLE_ATTR = 'title'
     SCHOOL_ATTR = 'school'
 
+    KINDERGARTEN = 'K'
+    VALID_GRADES = [KINDERGARTEN, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12].freeze
```

Comment:
We don't have ActiveRecord validation for `grade`; this code replicates client side validation found at:
https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/client/app/bundles/Teacher/components/accounts/shared/grade_level_section.tsx#L6